chpl/chpl-api/src/main/resources/log4j2.xml

```diff
@@ -374,18 +374,46 @@
                     interval="1" modulate="true" />
             </Policies>
         </RollingFile>
+        <RollingFile name="cacheListenerActivity"
```

Comment:
Just a note to add the logger to the other environments

Reply:
The cache logging config should be added to the other environments.